planner/core/src/main/java/eu/seaclouds/platform/planner/core/Planner.java

```diff
@@ -296,32 +309,31 @@ public SLAInfo() {}
 
     }
 
-    class OfferJson {
+    public static class Offering {
```

Comment:
extract those class into its own file. All the planner components may need to use this class instead of using String that needs to be parsed as `Offering` later